templates/base_index.html

```diff
@@ -28,9 +28,35 @@ <h1 style="font-weight: 100;">Ubuntu {{ releases.latest.short_version }} is here
     {% endblock notices_content %}
 
     <section class="p-strip is-deep is-bordered">
-      {% with gtm_event_label="ubuntu.com homepage" %}
-        {% include "shared/_insights_news_strip.html" %}
-      {% endwith %}
+      <div class="row p-divider">
+        <div class="col-9">
+          <h3>Latest news from our blog</h3>
+          <div id="latest-articles" class="row">
+            <div style="min-height: 9.1rem"><img src="https://assets.ubuntu.com/v1/2eb3f9f4-spinner-big.gif"></div>
```

Comment:
Missing alt on the image and the min-height could be quite large on small screens

Reply:
Please use Vanilla loading icon and animation:
```html
<i class="p-icon--spinner u-animation--spin">Loading...</i>
```

Reply:
Thanks I've used that Vanilla spinner.

But the min-height is because I want to try to prevent the page jumping. On large screens, this height is the same height as smallest the row will be (it might be bigger if some article titles flow onto 4 rows). On small screens, this min-height will be much less than the total height when the articles arrive, so it doesn't seem unreasonable to reserve this amount of space.